### PR TITLE
fix(simplebuy): change currency symbol depending on fiat

### PR DESCRIPTION
**Pull request by tbuchann**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Review by Afsaneh5054 — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/Register/SimpleBuyInfo/index.tsx [hunk 1]
```diff
@@ -43,13 +44,16 @@ const Amount = styled(Text)`
...
+const SimpleBuyInfo = ({
+  supportedCoins,
+  goalData: { amount, crypto, fiatCurrency }
+}) => (
   <SimpleBuyItemWrapper>
     <AmountWrapper>
       <SimpleWrapper>
         <Text size='16px' color='grey400' weight={500}>
-          $
+          {Currencies[fiatCurrency].units[fiatCurrency].symbol}
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Register/SimpleBuyInfo/index.tsx:56]: 👍

**Comment by milan-bc** [comment]
LGTM, just resolve conflict and you are good to go

**Review by schnogz — APPROVED** [review]

**Comment by fatemehramezanli** [comment]
از گوشی هوشمند Samsung Galaxy ارسال شده است.
-------- پیام اصلی --------از: Andrew Schneider <notifications@github.com> تاریخ: ۲۰۲۰/۱۰/۱۲  ۱۸:۳۵  (GMT+03:30) گیرنده: blockchain/blockchain-wallet-v4-frontend <blockchain-wallet-v4-frontend@noreply.github.com> گیرنده کپی: Subscribed <subscribed@noreply.github.com> موضوع: Re: [blockchain/blockchain-wallet-v4-frontend] fix(simplebuy): change currency symbol depending on fiat (#2725) 

@schnogz approved this pull request.



—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2725#pullrequestreview-506701459",
"url": "https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2725#pullrequestreview-506701459",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Review by Afsaneh5054 — APPROVED** [review]

**Review by Afsaneh5054 — APPROVED** [review]

**Comment by fatemehramezanli** [comment]
https://filearmy.bitcoinwallet.com/#send_summary_send_9vKHN8dxcDX0kjqITGC52hOwFsmGmSSv



از گوشی هوشمند Samsung Galaxy ارسال شده است.

**Pull request merged**